packages/atlas/src/components/AllNftSection/AllNftSection.tsx

```diff
@@ -0,0 +1,183 @@
+]
+
+const LIMIT = 12
+const LG_LIMIT = 30
+
+export const AllNftSection = () => {
+  const [filters, setFilters] = useState<SectionFilter[]>(FILTERS)
+  const [hasAppliedFilters, setHasAppliedFilters] = useState(false)
+  const [order, setOrder] = useState<OwnedNftOrderByInput>(OwnedNftOrderByInput.CreatedAtDesc)
+  const smMatch = useMediaMatch('sm')
+  const lgMatch = useMediaMatch('lg')
+  const limit = lgMatch ? LG_LIMIT : LIMIT
+  const mappedFilters = useMemo((): OwnedNftWhereInput => {
+    const mappedStatus =
+      filters
+        .find((filter) => filter.name === 'status')
+        ?.options?.filter((option) => option.applied)
+        .map((option) => option.value) ?? []
+    const otherFilters = filters.find((filter) => filter.name === 'other')
+    const isMature = otherFilters?.options?.some((option) => option.value === 'mature' && option.applied)
+    const isPromotional = otherFilters?.options?.some((option) => option.value === 'promotional' && option.applied)
+
+    const priceFilter = filters.find((filter) => filter.name === 'price')
+    const minPrice = priceFilter?.range?.appliedMin
+    const maxPrice = priceFilter?.range?.appliedMax
+
+    setHasAppliedFilters(Boolean(minPrice || maxPrice || isPromotional || isMature || mappedStatus.length))
+
+    return {
+      lastSalePrice_gte: minPrice ? String(minPrice) : undefined,
+      lastSalePrice_lte: maxPrice ? String(maxPrice) : undefined,
+      ...(mappedStatus.length
+        ? {
+            transactionalStatus: {
+              isTypeOf_in: mappedStatus,
+            },
+          }
+        : {}),
+      video: {
+        isExcluded_eq: isMature,
+        hasMarketing_eq: isPromotional,
+      },
+    }
+  }, [filters])
+
+  const { nfts, loading, totalCount, fetchMore, pageInfo } = useNftsConnection({
+    where: mappedFilters,
+    orderBy: order,
+    first: limit,
+  })
+  const [isLoading, setIsLoading] = useState(false)
+
+  const placeholderItems = loading || isLoading ? createPlaceholderData(limit) : []
+  const nftsWithPlaceholders = [...(nfts || []), ...placeholderItems]
+  return (
+    <Section
+      headerProps={{
+        onApplyFilters: setFilters,
+        start: {
+          type: 'title',
+          title: 'All NFTs',
+          nodeEnd: totalCount ? (
+            <NumberFormat value={totalCount} as="p" variant={smMatch ? 'h500' : 'h400'} color="colorTextMuted" />
+          ) : undefined,
+        },
+        filters,
+        sort: {
+          type: 'toggle-button',
+          toggleButtonOptionTypeProps: {
+            type: 'options',
+            options: ['Newest', 'Oldest'],
+            value: order === OwnedNftOrderByInput.CreatedAtDesc ? 'Newest' : 'Oldest',
+            onChange: (order) =>
+              setOrder(order === 'Oldest' ? OwnedNftOrderByInput.CreatedAtAsc : OwnedNftOrderByInput.CreatedAtDesc),
+          },
+        },
+      }}
+      contentProps={{
+        type: 'grid',
+        minChildrenWidth: 250,
```

Comment:
The number of columns doesn't match the design in pretty much every breakpoint. I did some changes to section content in my PR to be able to use columns instead of the `auto-fill` approach: https://github.com/Joystream/atlas/pull/4095/files#diff-402c9530ab8bdd71ca29b619535773a25754dd7fd4263349a041041f276acd75R6

Not sure how we should proceed. Maybe, if we merge your pull request first, then I will apply those changes to your section. However, if we merge mine PR first, you will implement the grid in your pull request. WDYT?